components/brave_ads/core/internal/targeting/behavioral/purchase_intent/purchase_intent_processor.h

```diff
@@ -24,6 +24,7 @@ namespace brave_ads {
 class PurchaseIntentResource;
 struct PurchaseIntentFunnelInfo;
 struct PurchaseIntentSignalInfo;
+struct TabInfo;
```

Comment:
Only the forward declaration (without the include) seems sufficient.

Reply:
Great catch. Thanks